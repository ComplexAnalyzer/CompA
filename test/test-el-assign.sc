def main() {
    a = [1,2,3,4];
    a = [1,2,3];
    b = a;
    print(b[1]);
    print(a[1]);
    b[0] = 100;
    print(a[0]);
    func(a);
    print(b[0]);
}


def func(a) {
    a[0] = 4;
}
