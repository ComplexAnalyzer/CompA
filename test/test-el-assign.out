2
2
100
4
